README.md

```diff
@@ -2,3 +2,50 @@
 
 [![Build Status](https://travis-ci.com/ZijiaZhang/CPSC436_Project.svg?branch=master)](https://travis-ci.com/ZijiaZhang/CPSC436_Project)
 [![codecov](https://codecov.io/gh/ZijiaZhang/CPSC436_Project/branch/master/graph/badge.svg)](https://codecov.io/gh/ZijiaZhang/CPSC436_Project)
+
+## Project Description: 
+This project aims to provide more precise people finder and chat environment. 
+It will allows users to make new friends easily. They will be able to find partners in the same lectures, 
+labs and also shout out to others to form a study group to prepare for their exams. 
```

Comment:
Our project aims to provide a friendly and focused environment for UBC students who are looking for friends, activities, or simply want to stay updated with what’s going on. Users can do things such as finding people who are in the same lecture or lab section as them, forming a study group for midterms or finals, looking for carpooling, etc.